miio/dreamevacuum_miot.py

```diff
@@ -56,9 +56,13 @@ class CleaningMode(Enum):
 
 class OperatingMode(Enum):
     Unknown = -1
+    Paused = 1
     Cleaning = 2
     GoCharging = 3
-    Paused = 14
```

Comment:
Should this (paused=14) still be kept (and the new Paused=1 renamed?) to avoid breaking the support for devices that report paused using "14"?

Reply:
```G:\Work\Code>miiocli dreamevacuummiot --ip 192.168.0.110 --token 54726c554e42716d5a31654661374c66 status

ERROR:miio.dreamevacuum_miot:Unknown OperatingMode (14)```
Right now my robot is sleeping on the dock, maybe, it means, that the current status is `Sleeping`?